mmv1/products/dns/Policy.yaml

```diff
@@ -31,6 +31,7 @@ timeouts:
   delete_minutes: 20
 custom_code:
   pre_delete: 'templates/terraform/pre_delete/detach_network.tmpl'
+  pre_create: 'templates/terraform/pre_create/dns_policy_enable_dns64_all_queries.go.tmpl'
```

Comment:
will also need pre_update (if supported) and post_read (to set the value from api read)